### PR TITLE
Only log the entire summary proto when confirm is false.

### PR DESCRIPTION
**Pull request by fejta**
Currently we are spamming our logs with the full content of every dashboard's summary every update loop.

### CODE REVIEW

File: pkg/summarizer/summary.go [hunk 1]
```diff
@@ -97,8 +97,8 @@ func Update(ctx context.Context, client gcs.Client, configPath gcs.Path, concurr
 					errCh <- errors.New(dash.Name)
 					continue
 				}
-				log.WithField("summary", sum).Info("summarized")
 				if !confirm {
+					log.WithField("summary", sum).Info("Summarized")
```
Comment (michelle192837) [inline pkg/summarizer/summary.go:101]: Should we still log _that_ we summarized the dashboard in either case, and the full summary only when !confirm?

Comment (fejta) [reply pkg/summarizer/summary.go:101]: Done

**Review by chases2 — APPROVED** [review]
/hold

**Pull request merged**